backend/client-pkg/services/ModelImportService.js

```diff
@@ -37,18 +26,23 @@ const sendRequest = (requestPath, responseCallback) => {
     },
     withCredentials: false
   }, (res) => {
-    let responseObject = null;
+    var response = null;
 
     res.on('data', (bodyChunk) => {
-    if(!responseObject)
-      responseObject = bodyChunk;
-    else
-      responseObject += bodyChunk;
+      if(!response)
+        response = bodyChunk;
+      else
+        response += bodyChunk;
     });
 
     res.on('end', () => {
       try {
-        responseCallback(JSON.parse(responseObject));
+        if (response.includes('error')) {
+          responseCallback(JSON.parse(response));
```

Comment:
i don't think you need to parse this here, you can just return a string if there is an error. Since we don't know what the errors look like (we didn't create the endpoint) we shouldn't expect them to be JSON.